Project_source/app/src/main/java/android/clicker/school_live_simulator/GuitarState.kt

```diff
@@ -0,0 +1,26 @@
+package android.clicker.school_live_simulator
+
+/**
+ * @author  Nikita Dimitryuk niksvob@mail.ru
+ */
```

Comment:
Add please version and version and since fields like other people did

Reply:
Fixed